docs/dev/table/connect.md

```diff
@@ -276,6 +276,40 @@ tables:
         type: VARCHAR
 {% endhighlight %}
 </div>
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- CREATE a 010 version Kafka table start from the earliest offset(as table source) and append mode(as table sink).
```

Comment:
Is this comment helpful? The other code examples also don't have this comment. We should make the example consistent to the others, also in terms of comments.

Reply:
I'm ok to remove it.